corehq/apps/change_feed/pillow.py

```diff
@@ -0,0 +1,55 @@
+import json
+from django.conf import settings
+from kafka import KafkaClient, KeyedProducer
+from casexml.apps.case.models import CommCareCase
+from corehq.apps.change_feed import data_sources
+from corehq.apps.change_feed.models import ChangeMeta
+from corehq.apps.change_feed.topics import get_topic
+from couchforms.models import all_known_formlike_doc_types
+from pillowtop.couchdb import CachedCouchDB
+from pillowtop.listener import PythonPillow
+
+
+class ChangeFeedPillow(PythonPillow):
+
+    def __init__(self, couch_db=None):
+        # todo: this will need to not be hard-coded if we ever split out forms and cases into their own domains
+        couch_db = couch_db or CachedCouchDB(CommCareCase.get_db().uri, readonly=False)
```

Comment:
No, not yet, but hopefully will be soon.

On 27 September 2015 at 21:05, Cory Zue <notifications@github.com> wrote:

> In corehq/apps/change_feed/pillow.py
> <https://github.com/dimagi/commcare-hq/pull/8329#discussion_r40512921>:
>
> > +from django.conf import settings
> > +from kafka import KafkaClient, KeyedProducer
> > +from casexml.apps.case.models import CommCareCase
> > +from corehq.apps.change_feed import data_sources
> > +from corehq.apps.change_feed.models import ChangeMeta
> > +from corehq.apps.change_feed.topics import get_topic
> > +from couchforms.models import all_known_formlike_doc_types
> > +from pillowtop.couchdb import CachedCouchDB
> > +from pillowtop.listener import PythonPillow
> > +
> > +
> > +class ChangeFeedPillow(PythonPillow):
> > +
> > +    def __init__(self, couch_db=None):
> > +        # todo: this will need to not be hard-coded if we ever split out forms and cases into their own domains
> > +        couch_db = couch_db or CachedCouchDB(CommCareCase.get_db().uri, readonly=False)
>
> are those officially moved over?!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8329/files#r40512921>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa